modules/ROOT/pages/acm-lightning-components.adoc

```diff
@@ -58,10 +58,42 @@ Card Border:: Specifies the card border thickness.
 Card Border Radius:: Specifies the card border radius. A larger radius makes the cards look more rounded.
 Card Shadow:: Specifies the shadow beneath each card.
 Message If Empty:: Shows a message if the carousel has no APIs. Default is `There are no APIs available right now.`
-Filter Condition (Advanced):: Limits the APIs shown in the carousel to those matching a standard Salesforce Object Query Language (SOQL) search. SOQL searches use package names, field names such as "API name" or "API description," and comparison of strings to the values of those fields.
+Filter Condition (Advanced):: Causes the carousel to show only APIs that match the given search condition, which uses the syntax described in the following section.
 API Details Button Label:: Enables you to set the API Details button label. Default is *API Details*.
 Learn More Button Label:: Enables you to set the Learn More button label. Default is *Learn More*.
 
+=== Filter Conditions
+
+The API Carousel component parameter *Filter Condition (Advanced)* causes the carousel to filter out the APIs that do not match the condition and show only the APIs that match. A condition is a standard search in Salesforce Object Query Language (SOQL).
+
+You can search these fields.
```

Comment:
```suggestion
You can search these fields:
```
GLOBALLY introduce lists, steps, syntax, and other graphic displays with a sentence or fragment that ends with a colon.

Reply:
Sorry about that. I mistakenly thought the style guide said the opposite. I'll remember this and introduce these with a colon from now on.